src/ethereum/rfc003/mod.rs

```diff
@@ -9,16 +9,24 @@ pub use ether_htlc::EtherHtlc;
 /// These are the hex-encoded ASCII-codepoints of the word "redeemed", padded to a length of 32 bytes.
 pub const REDEEMED_LOG_MSG: &str =
     "72656465656d6564000000000000000000000000000000000000000000000000";
+
 /// The log message emitted when the HTLC is refunded.
 ///
 /// These are the hex-encoded ASCII-codepoints of the word "refund", padded to a length of 32 bytes.
 pub const REFUNDED_LOG_MSG: &str =
     "726566756e646564000000000000000000000000000000000000000000000000";
+
 /// The return message emitted when someone attempted to refund the HTLC before the timeout.
 ///
 /// These are the hex-encoded ASCII-codepoints of the word "tooEarly", padded to a length of 32 bytes.
 pub const TOO_EARLY: &str = "746f6f4561726c79000000000000000000000000000000000000000000000000";
+
 /// The return message emitted when someone attempted to redeem the HTLC with an invalid secret.
 ///
 /// These are the hex-encoded ASCII-codepoints of the word "invalidSecret", padded to a length of 32 bytes.
 pub const INVALID_SECRET: &str = "696e76616c696453656372657400000000000000000000000000000000000000";
+
+/// The return message emitted by an ERC-20 token contract when a transfer is done, as per the ERC-20 EIP.
```

Comment:
Not 100% correct 😬 The function does not _return_ a message but it _triggers_ an event and leaves this log. 

```suggestion
/// The log message emitted by an ERC-20 token contract when a transfer is done, as per the ERC-20 EIP.
```

Reply:
Are the comments above valid? `return message emitted` This is what confused me. Shouldn't it be `message returned`?

Reply:
Yes, blame me and my poor english skills 😅

Either a transaction is a successful or unsuccessful:
1) in the success case, it can emit an event (LOG opcode) which writes a log message to the blockchain
2) in the failure case, every change on-chain would be rolled back, hence we can't emit an event (that would be a change to the blockchain). Hence we return this value from the function.